Zend/zend_compile.c

```diff
@@ -4936,6 +4983,62 @@ static zend_result zend_compile_func_sprintf(znode *result, zend_ast_list *args)
 	return SUCCESS;
 }
 
+typedef struct zend_special_func {
+	const char *name;
+	size_t name_len;
+	// zend_result (*handler)(znode *, zend_ast_list *);
+} zend_special_func;
+
+static const zend_special_func zend_special_func_compile_handlers[] = {
```

Comment:
There should be a comment with the definition of the explicitly listed functions, ie. what they need to meet and if not all are listed, why only smaller subset is listed.